src/app/components/burger-button/burger-button.component.ts

```diff
@@ -1,19 +1,21 @@
-import { Component, OnInit, Input, EventEmitter, Output } from '@angular/core';
+import { Component, EventEmitter, Output } from '@angular/core';
 
 @Component({
   selector: 'app-burger-button',
   templateUrl: './burger-button.component.html',
   styleUrls: ['./burger-button.component.scss'],
 })
-export class BurgerButtonComponent implements OnInit {
-  @Input() init = false;
+export class BurgerButtonComponent {
   @Output() opened = new EventEmitter<never>();
 
   active = false;
 
-  ngOnInit() {
-    this.active = this.init || false;
-  }
+  // On hold for further testing
```

Comment:
does the functionality work without the @Input ? Or with a parameter init rather than an input ? (no init property in parent components) if so remove dead code